hardcoded/hardcoded.go

```diff
@@ -214,8 +214,8 @@ var _bindata = map[string]*asset{
 			"\xf7\x6f\xab\xef\x11\xed\xdd\x8f\x87\x7c\x78\xd5\x41\xfa\x14\x94\x86\x22\x74\x8a\x2c\xa8\xbf\xd8" +
 			"\xde\xa2\x83\xa7\x69\x7c\x74\x96\xfc\x3f\x01\x00\x00\xff\xff",
 		size: 10915,
-		mode: 0664,
-		time: time.Unix(1544125033, 116862280),
+		mode: 0644,
+		time: time.Unix(1553712379, 443321183),
```

Comment:
there's a flag for `genbin` (I think `--nometadata`) that removes these timestamps